Modern/utilities/src/test/java/org/bonej/utilities/ElementUtilTest.java

```diff
@@ -175,14 +175,14 @@ public void testIsColorsBinary() {
 	public void testIsColorsBinaryBooleanTypeAssignable() {
 		final Img<BitType> img = ArrayImgs.bits(1);
 
-		final boolean isBinary = ElementUtil.isColorsBinary(img);
+		final boolean isBinary = ElementUtil.isBinary(img);
 
 		assertTrue("A BitType image should be binary", isBinary);
 	}
 
 	@Test(expected = NullPointerException.class)
 	public void testIsColorsBinaryThrowsNPEIfIntervalNull() {
-		final boolean result = ElementUtil.isColorsBinary(null);
+		final boolean result = ElementUtil.isBinary(null);
 
 		assertFalse("A null interval should not be binary color", result);
 	}
```

Comment:
I approve the change in method name for clarity and brevity.
In line with the change in method name, the assertion string should lose the "color"?